mission-control/docs/config-db/tutorials/file-scraper.md

```diff
@@ -0,0 +1,154 @@
+# Scraping Custom File
+
+In this tutorial we'll scrape a custom JSON file and see how config-db monitors changes to it over time.
+
+Let's start off by creating this example file that we want to scrape
+
+```json title="vms.json"
+{
+  "servers": [
+    {
+      "name": "web-server",
+      "ip": "192.168.1.100",
+      "port": 80,
+      "role": "frontend"
+    },
+    {
+      "name": "api-server",
+      "ip": "192.168.1.101",
+      "port": 8080,
+      "role": "backend"
+    },
+    {
+      "name": "database-server",
+      "ip": "192.168.1.102",
+      "port": 27017,
+      "role": "database"
+    }
+  ],
+  "logging": {
+    "level": "info",
+    "file_path": "/var/log/app.log"
+  }
+}
+```
+
+Place this file anywhere in your system and take note of its path. For the sake of simplicity, let's keep that file inside the `/tmp` dir.
```

Comment:
This is going to be hard to test for SaaS,  if concept is using config-db cli, then we should change this to a git repo source and using the SaaS

Reply:
Should I create a new example git repo so the users can fork it? Because to follow this tutorial, the user needs to modify the config file.

Reply:
New example created with the currency api